v2/manifests/deps/highcharts.yml

```diff
@@ -2,4 +2,9 @@ services:
   highcharts:
     image: onsdigital/highcharts-export-node
     ports:
-      - "9999:8080"
\ No newline at end of file
+      - "9999:8080"
+    healthcheck:
+      test: ["CMD", "curl", "-sSf", "http://localhost:8080/health"]
+      interval: ${HEALTHCHECK_INTERVAL:-30s}
+      timeout: 10s
+      retries: 10
```

Comment:
This presents a port clash with Babbage, no?

Reply:
No, the health check is run inside the container so it won't collide.